src/Client/V2/Service/EmsApiService.cs

```diff
@@ -233,7 +233,7 @@ private void InitializeAccessProperties()
             Assets = InitializeAccessClass<AssetsAccess>();
             Trajectories = InitializeAccessClass<TrajectoriesAccess>();
             Profiles = InitializeAccessClass<ProfilesAccess>();
-            ParameterSets = InitializeAccessClass<ParameterSetsAccess>();
+            AnalyticSets = InitializeAccessClass<AnalyticSetAccess>();

```

Comment:
It looks like this makes line 236 the same as 238. 

Maybe we leave the ParameterSets stuff in for now even though it's obsolete?

Otherwise I think we would need to remove the ParameterSets property completely since we never initialize it.